server/src/main/java/io/spine/server/entity/Repository.java

```diff
@@ -64,9 +64,10 @@
      * The {@link BoundedContext} to which the repository belongs.
      *
      * <p>This field is null when a repository is not {@linkplain
-     * BoundedContext#register(Repository) registered} yet.
+     * BoundedContext#register(Repository) registered} yet and
+     * after the repository is {@linkplain #close() closed}.
      */
-    private @MonotonicNonNull BoundedContext boundedContext;
```

Comment:
Not sure why this has changed to `@Nullable`.

Reply:
It's documented in the Javadoc of the field. We now clear the reference after the repository is closed.